docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+# Guidelines for CL editors
+
+## Naming terms
+
+### General rules
+1. All names should be singular nouns.
```

Comment:
```suggestion
1. All labels should be singular nouns.
```

Reply:
I agree that "label" or "labels" should be used throughout rather than "name" or "names", particularly as the annotation property used is 'label'.